configure

```diff
@@ -173,7 +173,7 @@ fi
 printf "Verifying Qt build environment ... "
 
 if [ -z "$QC_QTSELECT" ]; then
-	QC_QTSELECT="${QT_SELECT/qt/}"
+	QC_QTSELECT="$(echo $QT_SELECT | sed "s/qt//")"
```

Comment:
As far as I know `$QT_SELECT` might be equal to `qt4`, `qt5`, `4` or `5`. And other strings just will no work.